core/nwb.base.yaml

```diff
@@ -124,6 +124,15 @@ groups:
         acquisition system gain is 8000X, then the 'conversion' multiplier to get from
         raw data acquisition values to recorded volts is 2.5/32768/8000 = 9.5367e-9.
       required: false
+   - name: offset
+      dtype: float32
+      default_value: 0.0
+      doc: Scalar to add to the data after scaling by 'conversion' to finalize its coercion
+        to the specified 'unit'. Two common examples of this include (a) data stored in an
+        unsigned type that requires a shift after scaling to re-center the data,
+        and (b) specialized recording devices that naturally cause a scalar offset with
+        respect to the true units.
+      required: false
```

Comment:
Please add a note to new line 147 with "offset"

Reply:
Done, and added to all other objects I can find.